app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -633,5 +633,23 @@ def update_volumes(vm, attrs)
         vm.add_volume({:bootable => 'false', :quota => ovirt_quota, :blocking => api_version.to_f < 3.1}.merge(volume)) if volume[:id].blank?
       end
     end
+
+    def set_vm_interfaces_attributes(vm, vm_attrs)
+      if vm.respond_to?(:interfaces)
+        interfaces = vm.interfaces || []
+        vm_attrs[:interfaces_attributes] = interfaces.each_with_index.each_with_object({}) do |(interface, index), hsh|
```

Comment:
One question before merging: is empty hash a valid input? I mean in case there are no interfaces, you will have 
``` ruby
vm_attrs[:interfaces_attributes] = {}
```

Is it a valid input for ovirt?

Reply:
Yes, Ovirt can create a Vm without any nics